src/Hamiltonians/FroehlichPolaron.jl

```diff
@@ -0,0 +1,159 @@
+"""
+function FroehlichPolaron(
+    addr::OccupationNumberFS;
+    v=1.0,
+    mass=1.0,
+    omega=1.0,
+    l=1.0,
+    p=0.0,
+    momentum_cutoff=nothing,
+    mode_cutoff=100.0,
+)
```

Comment:
```suggestion
    function FroehlichPolaron(
        addr::OccupationNumberFS;
         v=1.0,
         mass=1.0,
        omega=1.0,
        l=1.0,
        p=0.0,
        momentum_cutoff=nothing,
        mode_cutoff=100.0,
    )
```
Sorry, not sure whether you wanted this looked at already, but I saw that you made some changes and I thought I might give some feedback.

For the function definition at the beginning of a docstring, if you indent by 4 spaces it will be marked as a code block, and rendered accordingly. How the documentation comes out formatted you can see on the preview page https://joachimbrand.github.io/Rimu.jl/previews/PR237/hamiltonians.html#Rimu.Hamiltonians.FroehlichPolaron